src/JJMasterData.Core/WebComponents/JJDataImp.cs

```diff
@@ -361,53 +317,48 @@ internal  DataImpDto GetCurrentProcess()
         return dto;
     }
 
-    private JJLinkButton GetBackButtonInstance()
+    private static JJLinkButton GetBackButton()
```

Comment:
Not Rider suggestion, by using static, you can use the method at the property initializer.

Reply:
The "Rider" always bugs me about this. But since I don't use the method outside the class I just ignore it